src/admin/server.rs

```diff
@@ -158,6 +158,37 @@ impl AdminServiceImpl {
         Ok(())
     }
 
+    pub async fn start_unit_on_vm(&self, vm: &str, unit: String) -> anyhow::Result<()> {
+        let vmname = format!("givc-{}.service", vm);
```

Comment:
Better to keep all formats in single place `utils::naming::*`

Reply:
Done.